youtube_dl/__init__.py

```diff
@@ -252,7 +252,7 @@ def parse_retries(retries):
             'key': 'EmbedThumbnail',
             'already_have_thumbnail': already_have_thumbnail
         })
-        if not already_have_thumbnail:
+        if not already_have_thumbnail and not opts.skip_download:
```

Comment:
This changes the behavior of the following use case:
1. Media was downloaded previously (file exists).
2. youtube-dl is run with `--embed-thumbnail --skip-download`.
3. No thumbnail is downloaded and embedded (it does both with current code).
Is this intentional to match `--embed-thumbnail --skip-download` behavior?